git_machete/tests/functional/test_machete.py

```diff
@@ -1381,3 +1552,235 @@ def test_update_with_fork_point_specified(self) -> None:
+        self.launch_command('github', 'anno-prs')
+        self.assert_command(
+            ["status"],
+            """
+            master
+            |
+            o-hotfix/add-trigger (diverged from origin)
+              |
+              o-ignore-trailing *  PR #3 (github_user) (diverged from & older than origin)
+
+            develop
+            |
+            x-allow-ownership-link  PR #7 (github_user) (ahead of origin)
+            | |
+            | x-build-chain (untracked)
+            |
+            o-call-ws  PR #31 (github_user) (ahead of origin)
+              |
+              x-drop-constraint (untracked)
+            """,
+        )
+
+    @mock.patch('git_machete.options.CommandLineOptions', FakeCommandLineOptions)
+    @mock.patch('urllib.request.urlopen', Opener)
+    @mock.patch('urllib.request.Request', GitAPIState)
+    def test_github_create_pr(self) -> None:
+        (
+            self.repo_sandbox.new_branch("root")
+                .commit("initial commit")
+                .new_branch("develop")
+                .commit("first commit")
+                .new_branch("allow-ownership-link")
+                .commit("Enable ownership links")
+                .push()
+                .new_branch("build-chain")
+                .commit("Build arbitrarily long chains of PRs")
+                .check_out("allow-ownership-link")
+                .commit("fixes")
+                .check_out("develop")
+                .commit("Other develop commit")
+                .push()
+                .new_branch("call-ws")
+                .commit("Call web service")
+                .commit("1st round of fixes")
+                .push()
+                .new_branch("drop-constraint")
+                .commit("Drop unneeded SQL constraints")
+                .check_out("call-ws")
+                .commit("2nd round of fixes")
+                .check_out("root")
+                .new_branch("master")
+                .commit("Master commit")
+                .push()
+                .new_branch("hotfix/add-trigger")
+                .commit("HOTFIX Add the trigger")
+                .push()
+                .commit_amend("HOTFIX Add the trigger (amended)")
+                .new_branch("ignore-trailing")
+                .commit("Ignore trailing data")
+                .sleep(1)
+                .commit_amend("Ignore trailing data (amended)")
+                .push()
+                .reset_to("ignore-trailing@{1}")
+                .delete_branch("root")
+                .new_branch('chore/fields')
+                .commit("remove outdated fields")
+                .add_remote('new_origin', 'https://github.com/user/repo.git')
+                .check_out("call-ws")
+        )
+        GitAPIState.pulls = [{'head': {'ref': 'ignore-trailing'}, 'user': {'login': 'github_user'}, 'base': {'ref': 'hotfix/add-trigger'}, 'number': '3', 'html_url': 'www.github.com'}]
+        self.launch_command("discover", "-y")
```

Comment:
Is this `-y` needed here if `FakeCommandLineOptions` already sets `self.opt_yes`.